jobs/launch_and_wait_for_job.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# this script creates a file test.log and prints the failure or success of the job as last line
+set -ex
+OUTPUT_DIR="$PWD"
+cd "$(dirname "$0")"
+
+source ./launch_job.sh "$@"
+# additionally these are set by the script above:
+# USER_TOKEN LAVA_TOKEN LAVA_JOB_NO
+
+echo "waiting for lava job ${LAVA_JOB_NO}"
+lavacli jobs wait ${LAVA_JOB_NO}
+echo "lava job ${LAVA_JOB_NO} completed with result:"
+
+# so that we create the log where we were called from
+cd $OUTPUT_DIR
+
+lavacli jobs logs ${LAVA_JOB_NO} > test.log
```

Comment:
It is actually possible to get the logs while the job is running rather than at the end. 
So maybe we can have 

```bash
lavacli jobs logs ${LAVA_JOB_NO} | sed -u -i "s/${USER_TOKEN}/==USER_TOKEN==/" | tee test.log
```

and then do `lavacli jobs wait`.